README.md

```diff
@@ -265,26 +286,65 @@ python runner.py --wf ttcom --executor dask/casa
+
+ - `--jobName`: Specify the desired condor job name. A dedicated folder will be generated, including all submission-related files.
+ - `--outputXrootdDir`: Indicate the XRootD directory's path (starting with `root://`) where the produced .coffea (and .root) files from each worker node will be transferred to.
+ - `--condorFileSize`: Define the number of files to process per condor job (default is 50). The input file list will be divided based on this count.
+ - `--remoteRepo` (optional, but recommended): Specify the path and branch of the remote repository to download the BTVNanoCommissioning repository. If not specified, the local directory will be packed and transferred as the condor input, potentially leading to higher loads for condor transfers. Use the format e.g. `--remoteRepo 'https://github.com/cms-btv-pog/BTVNanoCommissioning.git -b master'`.
+
+After executing the command, a new folder will be created, preparing the submission. Follow the on-screen instructions and utilize `condor_submit ...` to submit the jdl file. The output will be transferred to the designated XRootD destination.
 
-See [Submit standalone condor jobs](#submit-standalone-condor-jobs) section for details.
+<details><summary>Frequent issues for standalone condor jobs submission
+</summary>
+<p>
 
-## Make the json files
+1. CMS Connect provides a condor interface where one can submit jobs to all resources available in the CMS Global Pool. See [WorkBookCMSConnect Twiki](https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookCMSConnect#Requesting_different_Operative_S) for the instructions if you use it for the first time.
+2. The submitted jobs are of the kind which requires a proper setup of the X509 proxy, to use the XRootD service to access and store data. In the generated `.jdl` file, you may see a line configured for this purpose `use_x509userproxy = true`. If you have not submitted jobs of this kind on lxplus condor, we recommend you to add a line
+   ```bash
+   export X509_USER_PROXY=$HOME/x509up_u`id -u`
+   ```
+   to `.bashrc` and run it so the proxy file will be stored in your AFS folder instead of in your `/tmp/USERNAME` folder. For submission on cmsconnect, no specific action is required.
+
+</p>
+</details>
 
-Use `fetch.py` in folder `scripts/` to obtain your samples json files. `$input_DAS_list` is the name of your samples in CMS DAS, and $output_json_name$ is the name of your output samples json file.
+
+## Make the dataset json files
+
+Use `fetch.py` in folder `scripts/` to obtain your samples json files. You can create `$input_list` could be list of dataset name from DAS, and create the json contains `dataset_name:[filelist]`. For samples not publish in DAS could also access by specifying the path in the input list.
```

Comment:
could be list of dataset name from DAS, -> , which can be a list of datasets taken from CMS DAS,
 
 For samples not publish in DAS could also access by specifying the path in the input list. -> One can specify the local path in that input list for samples not published in CMS DAS.